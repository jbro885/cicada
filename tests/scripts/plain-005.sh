# test ENV inside command sup
echo $(echo foo bar | awk '{print $NF}')
echo $(echo foo bar z | awk '{print $NF}')
echo `echo foo bar1 | awk '{print $NF}'`
echo `echo foo bar z2 | awk '{print $NF}'`

VER1=`echo foo bar baz | awk '{print $NF}'`; echo $VER1
VER2=$(echo foo bar baz3 | awk '{print $NF}'); echo $VER2
VER3=$(echo foo bar baz4 | awk "{print $NF}"); echo $VER3
VER4=`echo foo bar baz5 | awk '{print $NF}'`; echo $VER4
VER5=`echo foo bar baz6 | awk "{print $NF}"`; echo $VER5
