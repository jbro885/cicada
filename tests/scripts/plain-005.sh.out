bar
z
bar1
z2
baz
baz3
foo bar baz4
baz5
foo bar baz6
